requirements.txt

```diff
@@ -4,4 +4,4 @@ nose==1.3.3
 requests==2.10.0
 celery==3.1.18
 newrelic
-git+https://github.com/yro/chunkey.git
+-e git+https://github.com/yro/chunkey.git@14f9c330d4ba5a4290dd0d17429b142618bc2c9f#egg=chunkey
```

Comment:
This seems like it might be dangerous. Should I make a release instead?

Reply:
This need to be double-checked whether it works for VEDA prod terraform plan.

Reply:
@yro @Qubad786 This should not cause any issue because `requirements.txt` is not used by terraform to install `edx-video-worker`